satellite/audit/service2.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+)
+
+// Service2 contains information for populating audit queue and processing audits.
+type Service2 struct {
```

Comment:
Let's call this a `Worker`.

Reply:
Do you still want us to name the individual processes `worker`s if we rename the main service to `Worker`?

Reply:
Let's make the main thing `Worker`, because I think it makes sense to have it top-level. It sits somewhere between a Service and a Chore -- so having a distinct name for it probably will help.

Reply:
Done